loadtest/client.go

```diff
@@ -21,26 +21,31 @@ func newClientFromToken(token string, serverUrl string) *model.Client4 {
 	return client
 }
 
-func loginAsUsers(cfg *LoadTestConfig) []string {
+func loginAsUsers(cfg *LoadTestConfig, entityStartNum int) []string {
 	tokens := make([]string, cfg.UserEntitiesConfiguration.NumActiveEntities)
 	r := rand.New(rand.NewSource(time.Now().UnixNano()))
 	order := r.Perm(cfg.LoadtestEnviromentConfig.NumUsers)
 
-	ThreadSplit(cfg.UserEntitiesConfiguration.NumActiveEntities, runtime.GOMAXPROCS(0)*2, PrintCounter, func(entityNum int) {
+	ThreadSplit(cfg.UserEntitiesConfiguration.NumActiveEntities, runtime.GOMAXPROCS(0)*2, PrintCounter, func(i int) {
 		// Add the usernum to start from
-		userNum := entityNum + cfg.UserEntitiesConfiguration.EntityStartNum
+		entityNum := i + entityStartNum
+		userNum := entityNum
 		client := model.NewAPIv4Client(cfg.ConnectionConfiguration.ServerURL)
 
-		// Random selection if picked
+		// Random selection if picked.
+		// TODO: Synchronize random selection across multiple loadtest instances by using
```

Comment:
I'm going to circle back on this, but wanted to get the meat of the changes up for review.

Reply:
I don't think this works now without screwing things up :P